INSTALL-ADVANCED.md

```diff
@@ -59,7 +59,7 @@ macOS:   | `/usr/local/lib/balena-cli/` <br> `/usr/local/bin/balena`
    [macOS](https://www.architectryan.com/2012/10/02/add-to-the-path-on-mac-os-x-mountain-lion/#.Uydjga1dXDg) |
    [Windows](https://www.computerhope.com/issues/ch000549.htm)
 
-> * If you are using macOS Catalina (10.15), [check this known issue and
+> * If you are using macOS 10.15 or later (Catalina, Big Sur), [check this known issue and
```

Comment:
This change is in its own commit. I have also updated issue #1479 to cover both Catalina and Big Sur.